docs/tutorial/cpp/src/from_one_to_multiple_tiles.md

```diff
@@ -18,7 +18,9 @@ First, we copy the tile data structure definition and paste it at top inside the
 {{#include ../../rust/src/main_multiple_tiles.rs:tile_data}}
 ```
 
-Next, we replace the *<span class="hljs-title">MainWindow</span> := { ... }* section at the bottom of the `memory.slint` file with the following snippet:
+We export the <span class="hljs-title">TileData</span>, so that we can use it from C++ later.
```

Comment:
Are you sure that export is needed? I shouldn't be, structs used in public properties are automatically exported.

Reply:
It is indeed not needed.  But i think we should recommend it anyway.